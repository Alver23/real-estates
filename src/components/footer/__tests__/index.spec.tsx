// Dependencies
import { shallow, ShallowWrapper } from 'enzyme';

// Under test file
import Footer from '../index';

describe('<Footer />', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Footer />);
  });

  it('should save a snapshot of the component', () => {
    expect(component).toMatchSnapshot();
  });
});
