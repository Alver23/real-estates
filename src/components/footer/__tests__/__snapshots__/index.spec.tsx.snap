// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should save a snapshot of the component 1`] = `
<footer
  className="absolute w-full bottom-0 flex justify-center items-center h-14 bg-green-300"
>
  <p
    className="text-white text-center text-sm"
  >
    Copyright Â© 
    2021
     All Rights.
  </p>
</footer>
`;
